packaging/fedora/globus-gsi-cert-utils.spec

```diff
@@ -3,7 +3,7 @@
 Name:		globus-gsi-cert-utils
 %global soname 0
```

Comment:
@msalle:
I believe we need to increase the `soname` value here as the interface of `globus_gsi_cert_utils_make_time()` changed slightly. Do you agree @ellert?

Reply:
I considered that too, but `ASN1_TIME` is more generic than `ASN1_UTCTIME` so it's still abi compatible. Furthermore AFAICT `ASN1_TIME *`  and `ASN1_UTCTIME *` are actually pointers to the same struct (see `/usr/include/openssl/ossl_typ.h`) ?

Reply:
But we can't know if this stays like that. IMHO if the interface changes types, that should be noted, even if they're the same internally, otherwise we would have to check that they stay the same for every new version of OpenSSL.

Reply:
I mean it might be unlikely, but what is the intention behind giving the same thing multiple names?

Reply:
Just to clarify: it's fine with me either way, I just don't know whether we should. So let's see what @ellert decides. And then one of you feel free to push the corresponding commit (I think you can) if necessary, or I'll do it later.

Btw note that this `soname` in the spec file is only used for SUSE-derived (i.e. when `suse_version` is non-zero). Why isn't that taken directly from the `configure.ac`, i.e. from `MAJOR_VERSION`, `MINOR_VERSION` and `AGE_VERSION`? (see https://github.com/pvanhoof/dir-examples#what-is-right for a very nice explanation of libtool and versioning).

Reply:
Resolved with https://github.com/gridcf/gct/pull/209#issuecomment-1399230962